makefile

```diff
@@ -0,0 +1,5 @@
+create_users:
+	go run main.go create_users
```

Comment:
Create one command for creating binary of this package and then these commands will use that binary with the command-line arguments.